plugin/src/main/groovy/com/btkelly/gnag/reporters/BaseExecutedViolationDetector.groovy

```diff
@@ -27,12 +27,12 @@ abstract class BaseExecutedViolationDetector extends BaseViolationDetector {
     abstract void executeReporter()
 
     protected final ReporterExtension reporterExtension
-    protected final ReportHelper reportHelper;
+    protected final ProjectHelper reportHelper;
```

Comment:
Please review codebase and change all instances of `reportHelper` to `projectHelper` :)